.vsts-dotnet-ci.yml

```diff
@@ -16,9 +16,11 @@ phases:
       Build_Debug_Intrinsics:
         _configuration: Debug-Intrinsics
         _config_short: DI
+        _includeBenchmarkData: false
```

Comment:
Why are we setting these parameters here and in the phase-template?  I think they only need to be set in the phase-template, right?

Reply:
This parameter needs to be passed for each configuration.
In phase-template we are definining the structure of different phases and some default values eg. everything for release and debug configuration has been defined there.
But for netfx and netcoreapp3.0, we have a different build configuration than debug and release,
hence has to pass those build confuguration specific variable